manifests/client.pp

```diff
@@ -0,0 +1,35 @@
+# @summary Configure backup client
+#
+# @param ssh_key_priv
+#   The private ssh key itself; generally a very long string...
+#
+# @param ssh_key_pub
+#   The public ssh key itself; generally a long string...
+#
+# @param ssh_key_type
+#   The encryption type used in the ssh key.
+#
+# @example
+#   include profile_backup::client
+class profile_backup::client (
+  String $ssh_key_priv,
+  String $ssh_key_pub,
+  String $ssh_key_type,
```

Comment:
suggest combine these into one param, let puppet code split into parts

Reply:
I disagree. We cannot leverage use of `ssh_authorized_key` later on if they are combined.